src/abi/stack.h

```diff
@@ -40,11 +40,43 @@ inline Index stackAlign(Index size) {
 // just local.get it anywhere there.
 inline void
 getStackSpace(Index local, Function* func, Index size, Module& wasm) {
-  auto* stackPointer =
-    GlobalUtils::getGlobalInitializedToImport(wasm, ENV, "STACKTOP");
```

Comment:
is there a reason to remove the old method? I think we can try the new code if the old code fails.

Reply:
Not a strong one.  My thinking was that Emscripten no longer generates the old idiom, and has not done so for some time now.  Unlike some other optmizations, --spill-pointers operates in conjunction with Emscripten and isn't applicable in general to WASM code generated by other means.   On the other hand, there is no reason in principle why a current build of wasm-opt could not be used with the output of an older Emscripten toolchain, or for other code generators to have adopted the stack convention used by Emscripten.  Anyhow, i've incorporated this and your other suggestions into the PR.